frontend/src/components/ui/AppDocumentUpload.vue

```diff
@@ -24,14 +31,21 @@
         </v-row>
       </div>
       <div v-if="uploadedDocuments.length > 0" class="mt-6 mx-4 mx-md-8 mx-lg-12">
-        <AppLabel>Uploaded Documents</AppLabel>
+        <AppLabel v-if="!documentType">Uploaded Documents</AppLabel>
```

Comment:
I think we should change "!documentType" to "!readonly" because I think the documentType is never empty.

Reply:
documentType is only applicable to application, i.e. not docs related to messages or organization. When a doc type is given we show a header and a slightly different layout (i.e. no table header "Upload Documents") otherwise repetitive on application screen.